dist/localforage.nopromises.js

```diff
@@ -718,7 +718,7 @@ return /******/ (function(modules) { // webpackBootstrap
 	            dbInfo.db = dbContext.db = db;
 	            self._dbInfo = dbInfo;
 	            // Share the final connection amongst related localForages.
-	            for (var k in forages) {
+	            for (var k = 0; k < forages.length; k++) {
```

Comment:
It's related to https://github.com/mozilla/localForage/commit/47aaaa82c7c86416e9e000973c5b105ca868e8cd